web-registry/src/pages/Dashboard/MyEcocredits/MyEcocredits.tsx

```diff
@@ -291,7 +291,7 @@ export const MyEcocredits = (): JSX.Element => {
                       icon: <AvailableCreditsIconAlt sx={sxs.arrow} />,
                       label: CREATE_SELL_ORDER_SHORT,
                       onClick: () => {
-                        track('sell1');
+                        track<'sell1'>('sell1');
```

Comment:
why aren't we adding metadata here?

Reply:
i have a commit with this that i haven't pushed yet